lib/chartmogul/csv/line_items/trial.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module ChartMogul
+  module CSV
+    module LineItems
+      # from https://chartmogul-samples.s3-eu-west-1.amazonaws.com/public/04_Invoice_line_items.csv
+      SUBSCRIPTION_HEADERS = ['Invoice external ID', 'External ID', 'Subscription external ID',
```

Comment:
is there a spec for this trial subscription? 
I might have missed it but I only saw some reformatting

Reply:
I don't think we need, it just inherit from subscription, same specs applies.